### PR TITLE
Handle cookies within request

### PR DESCRIPTION
**Pull request by raspbeguy**
Some websites needs cookies in order to work. With this modification, you can specify a `CookieJar` or subclass object to `py3.request` in order to exploit cookies.

### CODE REVIEW

File: py3status/request.py [hunk 1]
```diff
@@ -123,3 +129,20 @@ def headers(self):
...
+        """
+        try:
+            return self._cookiejar
+        except AttributeError:
+            return None
+
+    @cookiejar.setter
+    def cookiejar(self, cj):
+        """
+        Set the cookie jar in care we want to change it after object creation
```
Comment (tobes) [inline py3status/request.py]: s/care/case/

Comment (raspbeguy) [reply py3status/request.py]: Eeyep

**Comment by tobes** [comment]
I'd like things to be easier for the module writer maybe having CookieJar available via `self.py3` ping me when you've had a chance to do the updates you wanted

**Comment by raspbeguy** [comment]
@tobes I was thinking about adding a file `cookie.py` implementing our own CookieJar subclass, so that wouldn't change anything in this file (because `cookiejar` parameter accepts any CookieJar subclass already).

**Comment by tobes** [comment]
>  I was thinking about adding a file cookie.py

cool but module writers won't get access unless we give it to them.  modules should get everything they need via `self.py3` and not do `from py3status import ...` etc

**Comment by raspbeguy** [comment]
Yes of course I will make it available via `py3` but I mean, this could be the object of a new PR as it won't break the current version of the cookie feature.

**Comment by ultrabug** [comment]
I'm fine with this @raspbeguy and I agree that having it available through self.py3 would be great so please keep up the good work

**Pull request merged**